WooCommerce/Classes/Analytics/WooAnalyticsEvent.swift

```diff
@@ -87,3 +87,64 @@ extension WooAnalyticsEvent {
         WooAnalyticsEvent(statName: .featureFeedbackBanner, properties: ["context": context.rawValue, "action": action.rawValue])
     }
 }
+
+
+// MARK: - Issue Refund
+//
+extension WooAnalyticsEvent {
+    // Namespace
+    enum IssueRefund {
+        /// The state of the "refund shipping" button
+        public enum ShippingSwitchState: String {
```

Comment:
nit: `public` might not be necessary?
```suggestion
        enum ShippingSwitchState: String {
```